samples/SocialSample/Startup.cs

```diff
@@ -136,7 +136,7 @@ k web
                         JObject user = JObject.Parse(text);
 
                         var identity = new ClaimsIdentity(
-                            context.Options.AuthenticationType,
+                            context.Options.AuthenticationScheme,
```

Comment:
Ah yeah so when we smush multiple principals together that will be a problem. So we could add a special claim to every claims idenitity inside of the principal that marks the scheme that generated it